0.1 JS Fundamentals -  Introduction/0.1  JS Fundamentals - Introduction.js

```diff
@@ -0,0 +1,26 @@
+// Define a variable and assign a value
+let greeting = "Hello, world!";
+
+// Print the value of the variable to the console
+console.log(greeting);
```

Comment:
please log the output after each console.log()

Reply:
I have added all console log with their output